spec/workflow/intrinsic_function_spec.rb

```diff
@@ -450,6 +450,23 @@
       end
     end
 
+    describe "States.JsonMerge" do
+      let(:ctx) { {"context" => {"baz" => "qux"}} }
```

Comment:
This context is not needed, so it complicated the test - Can we remove it and just hardcode an empty context like we do for the other tests?